com.unity.render-pipelines.high-definition/Runtime/Material/LayeredLit/LayeredLit.shader

```diff
@@ -453,7 +453,7 @@ Shader "HDRP/LayeredLit"
     #pragma shader_feature_fragment _ENABLESPECULAROCCLUSION                     // Non-local
     #pragma shader_feature_fragment _ _SPECULAR_OCCLUSION_NONE _SPECULAR_OCCLUSION_FROM_BENT_NORMAL_MAP // Non-local
     #pragma shader_feature_raytracing _ENABLESPECULAROCCLUSION                     // Non-local
-    #pragma shader_feature_raytracing _ SPECULAR_OCCLUSION_NONE _SPECULAR_OCCLUSION_FROM_BENT_NORMAL_MAP // Non-local
+    #pragma shader_feature_raytracing _ _SPECULAR_OCCLUSION_NONE _SPECULAR_OCCLUSION_FROM_BENT_NORMAL_MAP // Non-local
```

Comment:
also mention it in upgrade guide (the rename)

Reply:
This is more a bugfix, so maybe it's better to mention it in the changelog no ? or both

Reply:
ah I see it is a typo for raytracing, ok don't care